packages/synapse-react-client/src/components/JSONArrayEditor/JSONArrayEditor.tsx

```diff
@@ -75,15 +94,19 @@ const JSONArrayEditor = React.forwardRef(function JSONArrayEditor(
     >
       <Form
         ref={ref}
-        schema={arraySchema}
+        schema={schema}
         className="JsonSchemaForm"
+        noHtml5Validate
         uiSchema={{
           'ui:submitButtonOptions': {
             norender: true,
           },
         }}
         validator={validator}
         formData={value}
+        formContext={{
+          allowRemovingLastItemInArray: true,
+        }}
```

Comment:
The annotations editor is heavily customized to not allow lists to be empty, so that the annotations field is always visible.

I want to re-use those components, but the user should be able to have an empty list here, so this is a new toggle so these lists can be made empty.

Reply:
Uploading Screen Recording 2023-11-20 at 10.34.06 AM.mov…